src/lib/mongodb.ts

```diff
@@ -14,6 +14,10 @@ declare global {
   }
 }
 
+const globalWithMongo = global as typeof globalThis & {
+  _mongoClientPromise: Promise<typeof MongoClient>
+}
+
```

Comment:
I'll admit I'm not sure I follow how this is different than the above on lines 7-15. I'm still fine adding it since it fixed the issue, but if someone has reasoning on what changed that made this required or how this and lines 7-15 are different it would be helpful to me.

Reply:
This actually looks like it addresses one of the linting errors that I mentioned previously, and addressed in [my announcement PR](https://github.com/USSF-ORBIT/ussf-portal-client/pull/669/files#diff-fce7fe8229577dcfe9a3f3db7e6093442fd9240dca70b4364c826bdc1abf1861). I paired with Suz on this, and I think it had something to do with a recent change to node's global typings.

Reply:
Does it make sense to merge this or just merge  yours since yours also addresses this? With regards to node's global typing, this was the top Google result https://stackoverflow.com/questions/68481686/type-typeof-globalthis-has-no-index-signature but again, all Greek to me, and a lot of it mentioned versions we weren't using. Wasn't sure if those changes got back-ported to 14.